src/App.tsx

```diff
@@ -1,14 +1,74 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { FilterBy } from './types/Filter';
+import { getTodos } from './api';
+
+const filteredTodos = (todos: Todo[], query: string, filterBy: FilterBy) => {
+  let preparedTodos = [...todos];
+
+  if (query) {
+    const preparedQuery = query.trim().toLowerCase();
+
+    preparedTodos = preparedTodos.filter(todo => todo.title.toLowerCase().includes(preparedQuery));
```

Comment:
you just need title here
```suggestion
    preparedTodos = preparedTodos.filter({ title }=> (
      title.toLowerCase().includes(preparedQuery)
    );
```